tfsec/exec/wrapper-tfsec.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash +e
+
+tfsec --format=checkstyle -e AWS045 | reviewdog -f=checkstyle -name="tfsec" -reporter=github-pr-review
```

Comment:
I think much more, like AWS045 have to be added, otherwise we will have a looot of warnings.

Reply:
@jportasa This is actually skipping `AWS045` - all the rest have been requested by Security.

Reply:
Crazy to follow what secOps say, I have sent them a message to reconsider this